tests/phpunit/data/blocks/fixtures/core__gallery__deprecated-2.server.html

```diff
@@ -0,0 +1,13 @@
+
+<ul class="wp-block-gallery columns-2 is-cropped alignwide wp-block-gallery-1">
+	<li class="blocks-gallery-item">
+		<figure>
+			<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==" data-id="1" alt="title" class="wp-image-1" />
+		</figure>
+	</li>
+	<li class="blocks-gallery-item">
+		<figure>
+			<img src="data:image/jpeg;base64,/9j/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/yQALCAABAAEBAREA/8wABgAQEAX/2gAIAQEAAD8A0s8g/9k=" data-id="2" alt="title" class="wp-image-2 is-layout-flex" />
```

Comment:
`is-layout-flex` doesn't look appropriate here as it's not on the other images. I am unsure if it has an effect.